pkg/metadata/host/host_notwin.go

```diff
@@ -0,0 +1,94 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+// +build !windows
+
+package host
+
+import (
+	"runtime"
+	"strings"
+
+	"github.com/DataDog/datadog-agent/pkg/util/cache"
+	"github.com/shirou/gopsutil/cpu"
+	"github.com/shirou/gopsutil/host"
+
+	log "github.com/cihub/seelog"
+)
+
+// Collect at init time
+var cpuInfo []cpu.InfoStat
+
+// InitHostMetadata initializes necessary CPU info
+func InitHostMetadata() error {
```

Comment:
same here: let's remove this and remove its usage from `cmd/agent/app/start.go`. The point of this method was to use grab the `cpuInfo` from WMI from the go code early in the Agent startup process, but this is useless now that we're not using WMI for that.

Reply:
It still prepopulates data that is used elsewhere... had a long discussion with @truthbk about this.  I believe it needs to stay.

Reply:
my question was mostly about why it’s important to initialize the CPU info at agent _startup_ time, instead of just using a wrapper that caches the info the first time it's called (i.e. what we do for a lot of other host metadata)

Not a blocker to merge though as I was just hoping we could clean up some old code. Let's re-discuss this next week with @truthbk 👍